src/main/java/spike/commands/DeleteTaskCommand.java

```diff
@@ -0,0 +1,60 @@
+package spike.commands;
+
+import spike.storage.TaskList;
+import spike.storage.Storage;
+import spike.ui.Ui;
+import spike.exceptions.SpikeException;
+import spike.tasks.Task;
+
+/**
+ * Represents a command to delete a task from the task list.
+ */
+public class DeleteTaskCommand extends Command {
```

Comment:
```suggestion
public class DeleteCommand extends Command {
```
This makes it consistent with your other commands e.g. FindCommand, ListCommand.